### ISSUE TITLE
Files with a different language than language existing entities throws an error while information extraction

### ISSUE THREAD

**Issue by mfacar** [huridocs/uwazi#4119]
Context: 
   - Uwazi launched with EXTERNAL_SERVICES true 
   - metadataExtraction feature is enabled
   - at least one template and property set for metadataExtraction
   - have labeled data
Steps to reproduce:
   - Upload a file in a different language than the available ones
   - Go to Metadata Extraction
   - Click review on the property
   Actual: There's an error since there is no entity in the queried language

**Comment by daneryl** [comment]
@mercy does this crashes the application ? is this being catched in the PR https://github.com/huridocs/uwazi/pull/4042 ?

**Comment by mfacar** [comment]
Yes @daneryl,  It was crashing the application but I added a catch for errors in response processing of the async tasks in PR #4042.

**Comment by aphilop** [comment]
@mfacar Is this still a problem?
I assume this is outdated and the issue can be closed. Let me know, thanks

**Comment by mfacar** [comment]
Yes @aphilop, it is outdated and can be closed. Actually, @Joao-vi is working on something related to suggestions for other file languages that are not installed in the instance.
